### PR TITLE
EZP-30376 [Travis] Implementing reporting tool to behat automated tests

### PR DESCRIPTION
**Pull request by m-tyrala**
[EZP-30376](https://jira.ez.no/browse/EZP-30376#)

- adding RSA key for sending Allure report to the reports server
- encrypting/decrypting key
- adding report processing script `ezreport`
- adding `after_script` section with test report processing

TODO before merge:
- [ ] squash commits
- [ ] remove temporary changes for testing related PRs

### CODE REVIEW

File: bin/ezbehat [hunk 1]
```diff
@@ -60,6 +60,7 @@ get_behat_features(){
     bin/behat ${PROFILE}${SUITE}${TAGS} --list-scenarios | awk '{ gsub(/:[0-9]+/,"",$1); print $1 }' | uniq -c | sort | awk '{ print $2 }'
 }
 
+
```
Comment (mnocon) [inline bin/ezbehat]: Please remove

File: bin/ezreport [hunk 2]
```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+# Examples of use:
+#
```
Comment (mnocon) [inline bin/ezreport]: Needs to be added/removed?

File: bin/ezreport [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+#
+
+SSH_KEY_PATH="$HOME/rsa_allure"
+STRICT='--strict'
```
Comment (mnocon) [inline bin/ezreport]: Is it used?

**Comment by m-tyrala** [comment]
@lserwatka this one can be merged

**Pull request merged**